en/tutorial-rest.md

```diff
@@ -17,7 +17,7 @@ In this tutorial, you will learn how to create a simple application that provide
 * `PUT` to update data
 * `DELETE` to delete data
 
-> **NOTE**: This is just a sample application. It lacks a lot of features such as authentication, authorization, sanitization of input and error management to name a few. Please use it as a building block for your application, or as a tutorial to understand how you can build a REST API with Phalcon. 
+> **NOTE**: This is just a sample application. It lacks a lot of features such as authentication, authorization, sanitization of input and error management to name a few. Please use it as a building block for your application, or as a tutorial to understand how you can build a REST API with Phalcon. You can also have a look at the [rest-api](https://github.com/phalcon/rest-api) project. 
```

Comment:
Can you add the alternative syntax for this? The one where we have the link at the bottom of the doc etc.

[ blah blah] [link stub here]

[link stub]: some URL